Android/app/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Apr 29 12:39:08 CEST 2019
```

Comment:
??

Reply:
It's not added from my end.

Reply:
it is

Reply:
#Mon Apr 29 12:39:08 CEST 2019
distributionBase=GRADLE_USER_HOME
distributionPath=wrapper/dists
zipStoreBase=GRADLE_USER_HOME
zipStorePath=wrapper/dists
distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip

This is what I found in master branch. Anyway I have removed that line. Please review it.

Reply:
My comment wasn't about the line, it was about the file. I mentioned in my *previous comments* also to remove this file.

Anyways, I made a commit to remove that file.